apps/api/v2/src/ee/me/outputs/me.output.ts

```diff
@@ -1,5 +1,11 @@
-import { IsInt, IsEmail, IsOptional, IsString } from "class-validator";
+import { Type } from "class-transformer";
+import { IsInt, IsEmail, IsOptional, IsString, ValidateNested } from "class-validator";
 
+export class MeOrgOutput {
```

Comment:
I don't see `MeOrgOutput` used elsewhere so we don't need to export it.

It's just me endpoint, but NIT would be that MeOrgOutput has id and `organizationId` is already on the me object.